clients/java/src/main/java/io/zeebe/client/impl/ZeebeClientCloudBuilderImpl.java

```diff
@@ -0,0 +1,196 @@
+    this.clientSecret = clientSecret;
+    return this;
+  }
+
+  @Override
+  public ZeebeClientBuilder brokerContactPoint(final String contactPoint) {
+    return innerBuilder.gatewayAddress(contactPoint);
+  }
+
+  @Override
+  public ZeebeClientBuilder gatewayAddress(final String gatewayAddress) {
+    return innerBuilder.gatewayAddress(gatewayAddress);
+  }
+
+  @Override
+  public ZeebeClientBuilder usePlaintext() {
+    return innerBuilder.usePlaintext();
+  }
+
+  @Override
+  public ZeebeClientBuilder credentialsProvider(final CredentialsProvider credentialsProvider) {
+    return innerBuilder.credentialsProvider(credentialsProvider);
+  }
+
+  @Override
+  public ZeebeClientBuilder withProperties(final Properties properties) {
+    return innerBuilder.withProperties(properties);
+  }
+
+  @Override
+  public ZeebeClientBuilder defaultJobWorkerMaxJobsActive(final int maxJobsActive) {
+    return innerBuilder.defaultJobWorkerMaxJobsActive(maxJobsActive);
+  }
+
+  @Override
+  public ZeebeClientBuilder numJobWorkerExecutionThreads(final int numThreads) {
+    return innerBuilder.numJobWorkerExecutionThreads(numThreads);
+  }
+
+  @Override
+  public ZeebeClientBuilder defaultJobWorkerName(final String workerName) {
+    return innerBuilder.defaultJobWorkerName(workerName);
+  }
+
+  @Override
+  public ZeebeClientBuilder defaultJobTimeout(final Duration timeout) {
+    return innerBuilder.defaultJobTimeout(timeout);
+  }
+
+  @Override
+  public ZeebeClientBuilder defaultJobPollInterval(final Duration pollInterval) {
+    return innerBuilder.defaultJobPollInterval(pollInterval);
+  }
+
+  @Override
+  public ZeebeClientBuilder defaultMessageTimeToLive(final Duration timeToLive) {
+    return innerBuilder.defaultMessageTimeToLive(timeToLive);
+  }
+
+  @Override
+  public ZeebeClientBuilder defaultRequestTimeout(final Duration requestTimeout) {
+    return innerBuilder.defaultRequestTimeout(requestTimeout);
+  }
+
+  @Override
+  public ZeebeClientBuilder caCertificatePath(final String certificatePath) {
+    return innerBuilder.caCertificatePath(certificatePath);
+  }
+
+  @Override
+  public ZeebeClientBuilder keepAlive(final Duration keepAlive) {
+    return innerBuilder.keepAlive(keepAlive);
+  }
+
+  @Override
+  public ZeebeClientBuilder withInterceptors(final ClientInterceptor... interceptor) {
+    return innerBuilder.withInterceptors(interceptor);
+  }
+
+  @Override
+  public ZeebeClientBuilder withJsonMapper(final JsonMapper jsonMapper) {
+    return innerBuilder.withJsonMapper(jsonMapper);
+  }
+
+  @Override
+  public ZeebeClient build() {
+    if (isNeedToSetCloudGatewayAddress()) {
+      ensureNotNull("cluster id", clusterId);
+      final String cloudGatewayAddress = String.format("%s.%s:443", clusterId, BASE_ADDRESS);
+      innerBuilder.gatewayAddress(cloudGatewayAddress);
+    } else {
+      Loggers.LOGGER.warn(
+          "The gateway address is overwritten. You are trying to connect *NOT* to the Camunda Cloud.");
```

Comment:
Please apply the [error guidelines](https://github.com/camunda-cloud/zeebe/wiki/Error-Guidelines) for this log. Also please consider the [logging guideline](https://github.com/camunda-cloud/zeebe/wiki/Logging) to use the best fitting log level. Debug level probably fits best, otherwise info.

Reply:
Done